src/main/java/core/basesyntax/impl/BalanceOperationHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.impl;

```

Comment:
why impl? better core.basesyntax.strategy.impl 
and the OperationHandler interface must be in the package core.basesyntax.strategy